.github/workflows/workflow_orders_on_merge.yml

```diff
@@ -16,6 +16,8 @@ on:
       - 'pyproject.toml'
 
 jobs:
+  call-lint:
```

Comment:
Do we want to run following ci jobs only after lint check is passed? If yes, please add 'needs' in all following jobs.